### DISCUSSION TITLE
Cant move objects with aninmations

### DISCUSSION THREAD

**Discussion by Sourlemn** [JacquesLucke/animation_nodes#1869] [Q&A]
So ive made this animation with a keyboard, but i decided to try and track it into real life footage. However when i set it all up the keyboard was way to small, and when i tried scaling it the objects that were animated didint move. Is there any way to force transform them or do i have to just give up?

https://user-images.githubusercontent.com/110069498/181129635-0767d530-2f63-45e8-8dba-8cf5727c6ccd.mp4

**Comment by OmarEmaraDev** [comment]
Seems like you just need to update the transformation ID keys of those objects after applying the necessary transformations.

**Reply by Sourlemn** [reply]
Thanks for the response, how do i do that? Ctrl-a all transforms messes everything up and it all like explodes and shit.

**Reply by Sourlemn** [reply]
Also i tried baking the animation using the set ketframes node and using the bake to keyframes button but it still couldnt handle transform, even if i use an empty to move it

**Comment by OmarEmaraDev** [comment]
I didn't mean applying in the sense of using the Apply Transforms operator.
I just meant you should transform your objects manually to the place you like, then update the ID key, by using the same operator you used to create the ID keys in the first place.
